README.md

```diff
@@ -33,33 +34,65 @@ To generate:
+To configure the iframe, sign in to [Masifunde's Fundraisingbox admin interface](https://secure.fundraisingbox.com) and go to `Einstellungen > Spendenformular` and select the relevant Spendenformular in the dropdown.
 
-## Style guide
+To fully customize the form, we hide most of the inputs in iframe, while rendering our own form fields outside of the iframe. We then [populate the iframe with these custom form field values](https://developer.fundraisingbox.com/v1.0/docs/form-prepopulation-api). This forces a reload and rerender of the iframe, which causes flickering. So we only do this when all custom form fields have values.
 
-Project uses ESLint AirBnb style guide without semicolons. The `precommit` git hook will
-automatically run ESLint to check if the code complies with the rules. If it fails then it will not
-push to the repo.
+### Einbettungsadresse (form embed url)
+The `Einbettungsadresse` should specify the full url of the page this form is used (e.g `https://www.masifunde.de/wie-sie-helfen/spenden`), and not just the host `https://www.masifunde.de`. 
+On a successful (or failed) donation attempt, the user gets redirected to the exact url in `Einbettungsadresse` with a `status` query param. 
+Then a `success` or `error` message is displayed in the iframe. 
+If the page doesn't have the Fundraisingbox iframe, no `success` or `error` message will be displayed.
+
+### iframe styling
+1. Sign in to [Masifunde's Fundraisingbox admin interface](https://secure.fundraisingbox.com).
+2. Then go to `Einstellungen > Spendenformular` and select the relevant Spendenformular in the dropdown.
+3. Scroll to the bottom of the `textarea` at the bottom of the page in the section `Design`.
+
+By checking the box for `Keine Standard-Stylesheets laden` the Fundraisingbox iframe will use the css written in the `textarea`.
+
+Note: This has to be standard css, *no scss or less*.
+
+For version control and backup purposes we also store this css in the file `Fundraisingbox_custom_css.css` in this repo. 
 
-### Contentful model naming convention
+### Testing Fundraisingbox locally
+
+Fundraisingbox only displays the form in the iframe if you load the iframe from a specified domain, e.g. `masifunde.netlify.com`. In order to test it locally you need to expose your localhost. You can use [ngrok](https://ngrok.com/) for this. Install ngrok and run the command ```ngrok http 3000 --region eu```. You will then see the address to which ngrok exposes your localhost.
+
+1. [Log into Masifunde's Fundraisingbox](https://secure.fundraisingbox.com).
+2. In the top right go to `Einstellungen > Spendenformular`.
+3. In the "Welches Formular wollen Sie bearbeiten" dropdown select the relevant Fundraising form.
+4. Paste your ngrok address into the field "Einbettungsadresse".
+5. Save.
+6. If you go to your ngrok address you should now be able to see the Fundraising form.
+
```

Comment:
Great stuff about FRB already, but would it help to add info about which form name corresponds to which form on the website?

Reply:
Yeah we can do that. Right now they're called "Development - spenden" and "Development - pate werden", but I guess I could just call them "Spenden" and "Pate werden" in the readme so we dont have to go back and change it later (which we probably will forget to do)